src/burn/engine/plan.cpp

```diff
@@ -1033,6 +1037,9 @@ extern "C" HRESULT PlanCachePackage(
 
     if (fBARequestedCache || NeedsCache(pPlan, pPackage))
     {
+        // The behavior for cache only mode is to do nothing on rollback, e.g. for subsequent install on demand scenarios.
```

Comment:
It's possible, yes, but part of the point. When we introduced this feature, it was to pre-cache the Windows Phone emulator (in the background, hence async originally coming in with this feature) so it's available when you want it later but may not be online. A bundle can still remove the cache even if its not installed, though.